src/test/java/com/conveyal/datatools/manager/jobs/NormalizeFieldTransformJobTest.java

```diff
@@ -127,7 +145,7 @@ public void canHandleInvalidSubstitutionPatterns() throws IOException {
                 new Substitution("\\Cir\\b", "Circle")
             )
         );
-        initializeFeedSource(transformation);
+        initializeFeedSource(TABLE_NAME, transformation);
```

Comment:
Would it make sense to inline `TABLE_NAME` or `FIELD_NAME`, or move their declarations to this method?